src/Chapter1.hs

```diff
@@ -620,29 +611,38 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
-
-
-{- |
-=💣= Task 10*
-
-You did it! You've passed all the challenges in your first training!
-Congratulations!
-Now, are you ready for the boss at the end of this training???
-
-Implement a function that returns the first digit of a given number.
-
->>> firstDigit 230
-2
->>> firstDigit 5623
-5
-
-You need to use recursion in this task. Feel free to return to it later, if you
-aren't ready for this boss yet!
--}
-
-firstDigit n = error "firstDigit: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n = (getLast n) + (secondLast n)
+  where
+    getLast :: Int -> Int
+    getLast number = mod number 10
+
+    secondLast :: Int -> Int
+    secondLast number = getLast (div number 10)
+
+-- |
+-- =💣= Task 10*
+--
+-- You did it! You've passed all the challenges in your first training!
+-- Congratulations!
+-- Now, are you ready for the boss at the end of this training???
+--
+-- Implement a function that returns the first digit of a given number.
+--
+-- >>> firstDigit 230
+-- 2
+-- >>> firstDigit 5623
+-- 5
+--
+-- You need to use recursion in this task. Feel free to return to it later, if you
+-- aren't ready for this boss yet!
+firstDigit :: Int -> Int
+firstDigit n
+  | n > 9 = firstDigit (dropLastDigit n)
+  | otherwise = n
```

Comment:
I see, that for a negative number like -19, this implementation will return the negative number itself instead of the last digit because the first check `n > 9` will always fail for negative numbers.

Reply:
I've added a case to handle negative numbers